HOW_WE_USE_GITHUB.md

```diff
@@ -119,27 +123,46 @@ Issues in the ["Sorting" tab of the project board][project-sorting] are consider
 - Determining if support can be provided for errors and questions
 - Closing out of any duplicate/off-topic issues
 
-The sorting engineers on rotation are not seeking to _resolve_ issues that arise. Instead, the goal is to understand the issue and to determine whether it is legitimate, and then to collect as much relevant information as possible so that the maintainers can make an informed decision about the appropriate resolution schedule.
+The core maintainers are not seeking to _resolve_ issues that arise. Instead,
+the goal is to understand the issue and to determine whether it is legitimate,
+and then to collect as much relevant information as possible so that the
+maintainers can make an informed decision about the appropriate resolution
+schedule.
 
 Issues will remain in the ["Sorting" tab][project-sorting] as long as the issue is in an investigatory phase (_e.g._, querying the user for more details, asking the user to attempt other workarounds, other debugging efforts, etc.) and are likely to remain in this state the longest, but should still be progressing over the course of 1-2 weeks.
 
 For more information on the sorting process, see [Issue Sorting Procedures](#issue-sorting-procedures).
 
 ### When do items move out of the "Sorting" tab?
 
-Items move out of the ["Sorting" tab][project-sorting] once the investigatory phase described in [What is done about the issues in the "Sorting" tab?](#what-is-done-about-the-issues-in-the-sorting-tab) has concluded and the sorting engineer has enough information to make a decision about the appropriate resolution schedule for the issue. The additional tabs in the project board that the issues can be moved to include the following:
+Items move out of the ["Sorting" tab][project-sorting] once the investigatory
+phase described in [What is done about the issues in the "Sorting"
+tab?](#what-is-done-about-the-issues-in-the-sorting-tab) has concluded and the
+core maintainer has enough information to make a decision about the
+appropriate resolution schedule for the issue. The additional tabs in the
+project board that the issues can be moved to include the following:
 
 - **"Support"** - Any issue in the ["Support" tab of the Planning board][project-support] is a request for support and is not a feature request or a bug report. Add the https://github.com/conda/infrastructure/labels/type%3A%3Asupport label to move an issue to this tab.
```

Comment:
It would be nice to have a seperate place for support type questions if possible. That way the issue tracker can just be for tracking issues and feature requests.